bundles/org.openhab.binding.sony/src/main/resources/ESH-INF/thing/scalar.xml

```diff
@@ -381,21 +381,21 @@
 	</channel-type>
 
 	<channel-type id="scalarsystemstoragesystemareacapacitymb" advanced="true">
-		<item-type>Number</item-type>
+		<item-type>Number:DataAmount</item-type>
 		<label>Storage System Capacity</label>
 		<description>The storage system capacity (in MB)</description>
 		<state readOnly="true" />
 	</channel-type>
 
 	<channel-type id="scalarsystemstoragetimesectofinalize" advanced="true">
-		<item-type>Number</item-type>
+		<item-type>Number:Time</item-type>
 		<label>Storage Time to Finalize</label>
 		<description>The time (in seconds) to finalize</description>
 		<state readOnly="true" />
```

Comment:
Since you are already providing the units as a QuantityType I think that describing that units are in seconds might be misleading for UIs that don't want to render it as such. (why that would be the case who knows, but it should still be up to the UI renderer to decide.)
Note: I don't know if my suggested description change here is appropriate so I'll leave it up to you to decide.

In addition, the UoM documentation suggests that a pattern attribute should be added as well.

Please make similar changes to all the other UoM channel types in your xmls.
```suggestion
		<description>The time until storage is finalized</description>
		<state readOnly="true" pattern="%d %unit%"/>
```

Reply:
Yep - forgot about the patterns!  Thanks